build.gradle

```diff
@@ -71,6 +74,10 @@ subprojects {
         checkTask.dependsOn('findbugs')
         checkTask.dependsOn('checkstyle')
 
+        def buildTask = tasks.findByName('build')
+        buildTask.dependsOn('dependencyUpdates')
```

Comment:
Can we don't do this? This task is so slow for me (it's useful though) that it's not worth it to run it on every build